src/PhpSpreadsheet/Helper/Migrator.php

```diff
@@ -248,18 +248,22 @@ public function getMapping()
     private function recursiveReplace($path)
     {
         $patterns = [
-            '/*.md',
```

Comment:
non-PHP files should not be removed, because it may contains documentation on how to use it and thus must be migrated too (this is how PhpSpreadsheet documentation was migrated).